perfTestUtils/dataStructures.go

```diff
@@ -31,31 +31,31 @@ const (
 // Config struct contains all values set by the config.xml file. Most, if not
 // all, can be overridden from command line.
 type Config struct {
-	APIName                              string  `xml:"apiName"`
-	TargetHost                           string  `xml:"targetHost"`
-	TargetPort                           string  `xml:"targetPort"`
-	NumIterations                        int     `xml:"numIterations"`
-	AllowablePeakMemoryVariance          float64 `xml:"allowablePeakMemoryVariance"`
-	AllowableServiceResponseTimeVariance float64 `xml:"allowableServiceResponseTimeVariance"`
-	TestCaseDir                          string  `xml:"testCaseDir"`
-	TestSuiteDir                         string  `xml:"testSuiteDir"`
-	BaseStatsOutputDir                   string  `xml:"baseStatsOutputDir"`
-	ReportOutputDir                      string  `xml:"reportOutputDir"`
-	ConcurrentUsers                      int     `xml:"concurrentUsers"`
-	TestSuite                            string  `xml:"testSuite"`
-	MemoryEndpoint                       string  `xml:"memoryEndpoint"`
-	RequestDelay                         int     `xml:"requestDelay"`
-	TPSFreq                              int     `xml:"TPSFreq"`
-	RampUsers                            int     `xml:"rampUsers"`
-	RampDelay                            int     `xml:"rampDelay"`
+	APIName                              string  `xml:"apiName" json:"apiName"`
```

Comment:
do we want omitempty on any fields that don't have a value? e.g. i imagine we don't want to save <targetHost></targetHost>

Reply:
Yes, omitempty should be set on any values which are not set via config file.